apps/web/src/pages/EmployeeProfile/components/CareerDevelopmentSection/Display.tsx

```diff
@@ -0,0 +1,201 @@
+          {intl.formatMessage({
+            defaultMessage:
+              'There are currently unanswered questions in this section. Use the "Edit" button to review and answer any relevant fields.',
+            id: "mMiZ3q",
+            description:
+              "Message for unanswered required questions in this section",
+          })}
+        </Well>
+      )}
+      <ToggleForm.FieldDisplay label={labels.moveInterest}>
+        {moveInterest ? (
+          <ul data-h2-list-style="base(none)" data-h2-padding="base(0)">
+            {unpackMaybes(careerDevelopmentOptions?.moveInterest).map((x) => {
+              const iconValue = moveInterests.includes(x.value);
+              return (
+                <li key={x.value}>
+                  <BoolCheckIcon value={iconValue}>
+                    {intl.formatMessage(getMoveInterest(x.value, iconValue))}
+                  </BoolCheckIcon>
+                </li>
+              );
+            })}
+          </ul>
+        ) : (
+          notProvided
+        )}
+      </ToggleForm.FieldDisplay>
+      <ToggleForm.FieldDisplay label={labels.organizationTypeInterest}>
+        {organizationTypeInterest ? (
+          <ul data-h2-list-style="base(none)" data-h2-padding="base(0)">
+            {unpackMaybes(
+              careerDevelopmentOptions?.organizationTypeInterest,
+            ).map((x) => {
+              const iconValue = organizationTypeInterests.includes(x.value);
+              return (
+                <li key={x.value}>
+                  <BoolCheckIcon value={iconValue}>
+                    {intl.formatMessage(
+                      getOrganizationTypeInterest(x.value, iconValue),
+                    )}
+                  </BoolCheckIcon>
+                </li>
+              );
+            })}
+          </ul>
+        ) : (
+          notProvided
+        )}
+      </ToggleForm.FieldDisplay>
+      <ToggleForm.FieldDisplay
+        hasError={!mentorshipStatus}
+        label={labels.mentorshipStatus}
+      >
+        {intl.formatMessage(displayMentorshipStatus(mentorshipStatus))}
+      </ToggleForm.FieldDisplay>
+      <ToggleForm.FieldDisplay label={labels.mentorshipInterest}>
+        {mentorshipInterest ? (
+          <ul data-h2-list-style="base(none)" data-h2-padding="base(0)">
+            {unpackMaybes(careerDevelopmentOptions?.mentorship).map((x) => {
+              const iconValue = mentorshipInterests.includes(x.value);
+              return (
+                <li key={x.value}>
+                  <BoolCheckIcon value={iconValue}>
+                    {intl.formatMessage(
+                      getMentorshipInterest(x.value, iconValue),
+                    )}
+                  </BoolCheckIcon>
+                </li>
+              );
+            })}
+          </ul>
+        ) : (
+          notProvided
+        )}
+      </ToggleForm.FieldDisplay>
+      <ToggleForm.FieldDisplay
+        hasError={!!execInterest !== execInterest}
```

Comment:
I could be wrong, but doesn't this always evaluate to the same thing? :sweat_smile:

Reply:
It's a little trick I saw recently :laughing:. ExecInterest type can be `execInterest: boolean | null | undefined`. If...
-  !!true === true
- !!false (or false) === false
- !!null (or false)  === null
- !!undefined (or false) === undefined

Since it's only false if it's null or undefined the little trick above works.
I forgot to revert this back to `empty(execInterest)` haha

Reply:
Fixed in [81b268c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12680/commits/81b268cfbfc291a2819e389ed49ff11c60235864)